extras/create_snapshots_reports_scorecard.py

```diff
@@ -684,11 +690,12 @@ def create_third_party_snapshots(db, cyhy_db_section, third_party_report_ids):
         else:
             failed_tp_snaps.append(third_party_id)
 
+    time_to_generate_tp_snaps = time.time() - all_tps_start_time
     logging.info(
         "Time to create all grouping node and third-party snapshots:"
-        " {:.2f} minutes".format(round(time.time() - all_tps_start_time, 1) / 60)
+        " %.2f minutes", time_to_generate_tp_snaps / 60
```

Comment:
I'm not 100% sure this will actually work because I set up the suggestion manually but I see no reason not to combine the strings with the other changes.

```suggestion
        "Time to create all grouping node and third-party snapshots: %.2f minutes",
        time_to_generate_tp_snaps / 60
```

Reply:
I wasn't able to accept your suggestions in the GitHub web GUI, due to this error: `Applying suggestions on deleted lines is currently not supported.` but I did make them manually in https://github.com/cisagov/cyhy-reports/pull/117/commits/11e6218aa7f93cd2aca3bbf52a8ec76b39b35729.